frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListHeaders/uploadListHeaders.component.tsx

```diff
@@ -17,28 +17,35 @@
 
 import { DashboardListHeaderLabel } from '@components/dashboard/dashboardList';
 import { FormattedMessage } from 'react-intl';
-import { UploadsListHeader } from './uploadFileForm.styles';
+import { DashboardListHeaderContainer } from '@components/dashboard/dashboardList/dashboardListHeader/dashboardListHeader.styles';
+import { SortingDirection } from '@components/dashboard/dashboardList/dashboardList.types';
+import { FilenameLabelWrapper } from './uploadListHeaders.styles';
+
+type IUploadListHeaders = {
+	onClickFilenameLabel: () => void;
+	isUploading: boolean;
+	sortingDirection: SortingDirection;
+};
 
 export const UploadListHeaders = ({
-	defaultSortConfig,
-	setSortConfig,
+	onClickFilenameLabel,
 	isUploading,
-}) => (
-	<UploadsListHeader
-		onSortingChange={setSortConfig}
-		defaultSortConfig={defaultSortConfig}
-	>
-		<DashboardListHeaderLabel key="file" name="file.name" minWidth={122}>
-			<FormattedMessage id="uploads.list.header.filename" defaultMessage="Filename" />
+	sortingDirection,
+}: IUploadListHeaders) => (
+	<DashboardListHeaderContainer>
+		<DashboardListHeaderLabel name="file.name" minWidth={122} sortingDirection={sortingDirection} sort>
+			<FilenameLabelWrapper onClick={onClickFilenameLabel}>
+				<FormattedMessage id="uploads.list.header.filename" defaultMessage="Filename" />
+			</FilenameLabelWrapper>
 		</DashboardListHeaderLabel>
-		<DashboardListHeaderLabel key="destination" width={352}>
+		<DashboardListHeaderLabel width={352}>
 			<FormattedMessage id="uploads.list.header.destination" defaultMessage="Destination" />
 		</DashboardListHeaderLabel>
-		<DashboardListHeaderLabel key="revisionName" width={isUploading ? 359 : 399}>
+		<DashboardListHeaderLabel width={isUploading ? 359 : 399}>
 			<FormattedMessage id="uploads.list.header.revisionName" defaultMessage="Revision Name" />
 		</DashboardListHeaderLabel>
-		<DashboardListHeaderLabel key="progress" width={297} hidden={!isUploading}>
+		<DashboardListHeaderLabel width={297} hidden={!isUploading}>
 			<FormattedMessage id="uploads.list.header.progress" defaultMessage="Upload Progress" />
 		</DashboardListHeaderLabel>
-	</UploadsListHeader>
+	</DashboardListHeaderContainer>
```

Comment:
**NOT a change request**
I suppose if the styled components defined in a _styled.ts_ file are only used inside the respective _component.tsx_ file, they can strip away the prefix. e.g.
`DashboardListHeaderContainer -> Container`
`DashboardListHeaderLabel -> Label`
`FilenameLabelWrapper -> Filename`

This way the code looks easier to read (imo)